R/axis.R

```diff
@@ -88,14 +88,23 @@ axis.gsplot <- function(object, ..., n.minor=0, tcl.minor=0.15, reverse=NULL) {
   
 }
 
-draw_axis <- function(axis.args){
+draw_axis <- function(object, side.name){
+  axis.args <- object[[side.name]][['axis']]
+  side.lim <- object[[side.name]][['lim']]
+
+  if(all(is.na(side.lim)) || is.numeric(side.lim)){
+    fun.name <- 'axis'
+  } else {
+    fun.name <- paste0('axis.', class(side.lim)[1L])
+    axis.args$at <- get_axTicks(object, as.side(side.name))
```

Comment:
can you remove this line? What does it do? I think you want to let the function `axis.class` figure out the `at`